src/docs/Foundations/Typography/HeadingStyles/index.mdx

```diff
@@ -1,49 +1,114 @@
 ---
 title: 'Heading styles'
 order: 5
+description: 'The heading pattern provides basic typographic styles for your titles to help the user quickly scan main informations in the different section of your page'
 status:
   sketch: 'wip'
   scss: 'wip'
 ---
 
-> The heading patterns provide basic typographic styles for headings.
+> The heading pattern provides basic typographic styles for your titles to help the user quickly scan main informations in the different section of your page.
 
-## Usage
+There are two different types of heading styles for Mozaic. One is a **text only** and the other style is **a combination of a text and an underline**.
 
-They are destined to style mono-line (or few lines) headings and help the user to quickly scan major informations of the page or section, and to create hierarchy between them.
+_text heading only & underlined heading_
 
-Heading styles are patterns meant to help you use consistent typographic hierarchy with the context and across responsive viewports. It's mainly reserved for users wishing to focus on tasks with added value and user experience. Text classes are families of several font-sizes and help create a hierarchy of contents.
+## Text headings
 
-## Headings styles define three properties of a text :
+There are three different font-weights available for your text headings.
 
-- Font-size _(that may vary depending on the viewport)_
-- Line-height _(that are optimised for readability and accessibility)_
-- Font-weight _(regular or semi-bold)_
+| Font-weight | Usage                                                                |
+| ----------- | -------------------------------------------------------------------- |
+| Light       | This font-weight must be used for bigger font-size                   |
+| Regular     | Use this font-weight for your common text headings                   |
+| SemiBold    | This font-weight allows you to visually emphasise your text headings |
```

Comment:
This part can be moved after the alignment section.

Replace ~Text headings~ by `Font weight`